### ISSUE TITLE
window-rules broken?

### ISSUE THREAD

**Issue by myfreeweb** [WayfireWM/wayfire#373]
I've had to disable window-rules after the new config thing landed. I don't use it, but had it enabled for no reason, caught a crash:

```
* thread #1, name = 'wayfire', stop reason = signal SIGSEGV
  * frame #0: 0x00000008005daf9f libwf-config.so.0`wf::config::section_t::get_registered_options() const + 31
    frame #1: 0x000000080d602c14 libwindow-rules.so`wayfire_window_rules::init() + 148
    frame #2: 0x000000000030320f wayfire`plugin_manager::reload_dynamic_plugins() + 1087
    frame #3: 0x0000000000302c89 wayfire`plugin_manager::plugin_manager(wf::output_t*) + 265
    frame #4: 0x0000000000306830 wayfire`wf::output_impl_t::output_impl_t(wlr_output*) + 128
    frame #5: 0x00000000002be891 wayfire`wf::output_layout_output_t::ensure_wayfire_output() + 49
    frame #6: 0x00000000002ba408 wayfire`wf::output_layout_output_t::apply_state(wf::output_state_t const&, bool) + 408
    frame #7: 0x00000000002b9cec wayfire`wf::output_layout_t::impl::apply_configuration(std::__1::map<wlr_output*, wf::output_state_t, std::__1::less<wlr_output*>, std::__1::allocator<std::__1::pair<wlr_output* const, wf::output_state_t> > > const&) + 1116
    frame #8: 0x00000000002b7d24 wayfire`wf::output_layout_t::impl::reconfigure_from_config() + 1172
    frame #9: 0x00000000002b7434 wayfire`wf::output_layout_t::impl::add_output(wlr_output*) + 468
    frame #10: 0x00000000002b4af3 wayfire`handle_wrapped_listener(wl_listener*, void*) + 35
    frame #11: 0x000000080042e55f libwlroots.so.4`wlr_signal_emit_safe(signal=0x0000000802326348, data=0x000000080c6b4e00) at signal.c:29
    frame #12: 0x00000008003dafbe libwlroots.so.4`new_output_reemit(listener=0x000000080c518d58, data=0x000000080c6b4e00) at backend.c:143
    frame #13: 0x000000080042e55f libwlroots.so.4`wlr_signal_emit_safe(signal=0x000000080234d728, data=0x000000080c6b4e00) at signal.c:29
    frame #14: 0x00000008003cff71 libwlroots.so.4`scan_drm_connectors(drm=0x000000080234d700) at drm.c:1486
    frame #15: 0x00000008003cc7e2 libwlroots.so.4`backend_start(backend=0x000000080234d700) at backend.c:26
… 
```

**Comment by ammen99** [comment]
We were just discussing this on IRC, we're missing a XML file for it.

**Comment by ammen99** [comment]
Fixed in e448918624b4b14561e919189a64b5e61c7d3bbf
